tools/prepare_release.cmd

```diff
@@ -1 +1 @@
-@powershell -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "%~dp0\tools\run-csx.ps1 %~dp0\tools\prepare-release.csx %*"
```

Comment:
This was never tested and probably didn't work, because the path was wrong... I broke it in https://github.com/FakeItEasy/FakeItEasy/pull/1410